pkgs/applications/editors/jetbrains/common.nix

```
@@ -1,14 +1,20 @@
 { stdenv, lib, makeDesktopItem, makeWrapper, patchelf
 , coreutils, gnugrep, which, git, unzip, libsecret, libnotify
+, buildFHSUserEnv
 }:
 
-{ name, product, version, src, wmClass, jdk, meta }:
+{ name, product, version, src, wmClass, jdk, meta, fhs ? null }:
 
 with stdenv.lib;
 
 let loName = toLower product;
     hiName = toUpper product;
     execName = concatStringsSep "-" (init (splitString "-" name));
+    wrap = if (fhs != null) then (buildFHSUserEnv {
+      name = "${loName}-fhs";
+      targetPkgs = pkgs: fhs;
+      runScript = ''/bin/sh'';
+    }) else null;
```

```suggestion
    wrap = lib.optional (fhs != null) (buildFHSUserEnv {
      name = "${loName}-fhs";
      targetPkgs = pkgs: fhs;
      runScript = ''/bin/sh'';
    });
```

Did not check if this works but it is nicer instead of if else.